### PR TITLE
docs: add zhengcaiyun-uses-apisix.md

### PR DESCRIPTION
**Pull request by saltism**

### CODE REVIEW

**Review by Yilialinn — APPROVED** [review]

File: blog/en/blog/2023/12/26/zhengcaiyun-uses-apisix.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+* **Declines in monitoring metrics such as response time (RT) and throughput:** We also employed the Prometheus-formatted BRAF interface for monitoring. On this interface, certain response volume and throughput curves exhibited a noticeable downward trend.
+
+The pressure in these two aspects gradually manifested as we transitioned historical solutions into the "Highway" project, coupled with the increasing traffic volume.
+
+In order to ensure the stability of our business operations and enhance the overall user experience, we have implemented a range of optimization strategies. Broadly speaking, our efforts have centered around three key approaches:
+
+![Optimization_solutions](https://static.apiseven.com/uploads/2023/12/26/Bs0op31X_3.jpg)
```
Comment (GeekEmily) [inline blog/en/blog/2023/12/26/zhengcaiyun-uses-apisix.md]: "barrel model" in the chart seems to be an incorrect phrase; it doesn't match the phrase you use in the paragraph below either

File: blog/en/blog/2023/12/26/zhengcaiyun-uses-apisix.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+Building on the insights gained from working with the Dubbo2 protocol, our focus now shifts to driving the upgrade of the Triple protocol to tackle performance issues.
+
+### Validating Performance through Stress Testing
+
+Our architecture has undergone comprehensive stress testing, providing us with a clear understanding of existing bottlenecks. Moving forward, our commitment extends to continuous testing, delving deeper into the untapped performance potential of this foundational infrastructure.
+
+### Charting the Course for the Future High-Speed Highway Architecture
+
+![Highway_future](https://static.apiseven.com/uploads/2023/12/26/r0AMeygL_9.jpg)
```
Comment (GeekEmily) [inline blog/en/blog/2023/12/26/zhengcaiyun-uses-apisix.md]: The shown text should be APISIX, instead of Apisix

**Review by GeekEmily — CHANGES_REQUESTED** [review]
A thorough check for terminologies is needed

**Comment by yzeng25** [comment]
@JYan00 any update?

**Review by yzeng25 — APPROVED** [review]

**Review by guoqqqi — APPROVED** [review]

**Pull request merged**